### PR TITLE
Zombie tweaks + Sewer Gator + Fungal Ant

### PR DESCRIPTION
**Pull request by pisskop**
Tweaks several small things, the largest being

-Shocker Brute more damaging, less healthiful.  More in line as a brute than a damage sponge with lightning
-Adds Sewer Gator.  Fairly high armor, high attack
-Zombie kids evolve quicker
-Corrosive zombie encouraged to attack more and spit slightly less
-Fungal Ant puffs spores on death to infect nearby ants.
-Fungal zombies have their sight range reduced, but can actually 'see' now instead of having stats they dont use (sight radius without sight)

### CODE REVIEW

**Comment by Coolthulhu** [comment]
Most of those look good, but:
* Corrosive zombie is supposed to spit, not hit. Spitting is its primary attack that prevents it from being kited trivially using a cheap wooden spear
* Swimmer zombie shouldn't evolve into biter because they have different equipment sets

**Comment by pisskop** [comment]
Should biters have swimmer sets?

They always seem more like man-alligators than man-wolves

**Comment by kevingranade** [comment]
Also nothing wrong get with having a water based biter and a land based
biter.

**Comment by Coolthulhu** [comment]
Just make sure that the equipment sets aren't changed by evolutions.

We should probably change the evolution system to pass the initial equipment set somehow. That would make it much more flexible.

**Comment by Coolthulhu** [comment]
Json error here. Check out the jenkins' output.

**Comment by pisskop** [comment]
Nah.

jenkins is getting funky with how it seems to be reading it.

File: data/json/monsters.json [hunk 1]
```diff
@@ -4325,6 +4369,47 @@
...
+            ]
+        }, ["GRAB", 30], ["GRAB_DRAG", 10]],
+    "description":"In the late 20th century there was an urban legend about pet alligators getting flushed down the toilet and growing to adulthood in sewers.  This large specimen doesn't look like its used to humans as anything other than a meal.",
+    "flags":["SEES", "HEARS", "GOODHEARING", "SMELLS", "KEENNOSE", "ANIMAL", "BLEED", "ATTACKMON", "BONES", "LEATHER", "SWIMS"],
+    "anger_triggers":["PLAYER_CLOSE", "PLAYER_WEAK"],
+    "placate_triggers":["MEAT"],
+    "fear_triggers": ["FIRE", "HURT"],
+    "categories":["WILDLIFE"]
+    },{
+    "type" : "MONSTER",
```
Comment (pisskop) [inline data/json/monsters.json]: Right here.

Jenkins is reading this as an error, if Im reading this correctly.

Something something using a seperator on the last line, when in fact its the start of a new critter

Comment (Zireael07) [inline data/json/monsters.json]: line 4404 looks suspicious, what's that } bracket doing there?

**Comment by pisskop** [comment]
Thats the close bracket for the special attack for the special attack 'bite'

**Comment by pisskop** [comment]
. . . >.>

File: data/json/monsters.json [hunk 2]
```diff
@@ -2820,8 +2824,48 @@
...
+    "melee_skill":4,
+    "melee_dice":3,
+    "melee_dice_sides":4,
+    "melee_cut":8,
+    "dodge":2,
+    "armor_bash":4,
+    "armor_cut":6,
+    "vision_day":25,
+    "vision_night":5,
+    "death_drops": [ "mon_zombie_swimmer_death_drops", 65 ],
```
Comment (Coolthulhu) [inline data/json/monsters.json]: This one is not correct, causes an error on load.

Test your PRs at least once.

**Pull request closed without merging**